src/test/java/io/strimzi/kafka/bridge/http/ConsumerSubscriptionTest.java

```diff
@@ -214,5 +216,7 @@ void listConsumerSubscriptions(VertxTestContext context) throws InterruptedExcep
                 });
 
         assertTrue(context.awaitCompletion(TEST_TIMEOUT, TimeUnit.SECONDS));
+        consumerService()
```

Comment:
It seems that we are deleting consumer in each test case or am i wrong ? Could we specify this in @AfterEach notation to reduce some code lines ? If it is problem now to reworked that you can create issue. What do you think ?

Reply:
The problem is that consumers don't have always the same name so how to deal with it in a common @aftereach code? How to deal with it? @granzoto changed the consumer names in this PR to have more detailed information for each test.